USAGE.md

```diff
@@ -0,0 +1,138 @@
+# Usage
+
+## Creating a Connect API Client
+
+There are two methods available for creating a client:
+
+- `new_client_from_environment`: Builds a new client for interacting with 1Password Connect using the `OP_CONNECT_TOKEN` and `OP_CONNECT_HOST` _environment variables_.
```

Comment:
Do we want to italicize the "environment variables" string here? I would assume not?

```suggestion
- `new_client_from_environment`: Builds a new client for interacting with 1Password Connect using the `OP_CONNECT_TOKEN` and `OP_CONNECT_HOST` environment variables.
```

Reply:
Yeah, we don't have to. Nice catch.